### DISCUSSION TITLE
How to View Scalene JSON File

### DISCUSSION THREAD

**Discussion by garrettwrong** [plasma-umass/scalene#398] [Q&A] [answered]
Hi, I have profiled some code and output a JSON file on a remote machine (behind strict firewall) and without a GUI.  After transferring to a local machine, how can I open this file (or convert it to the html)?

I tried the website http://plasma-umass.org/scalene-gui/, but it doesn't seem to do anything after I select my file.  (Maybe it is too large at ~5MB).

Thanks!

**Comment by emeryberger** [comment, marked as answer]
Newer versions of Scalene take steps to ensure that the JSON file never gets so large - please try again!

**Comment by garrettwrong** [comment]
Sure, I'll try the next time I intend to profile something and report back. Thanks!
